stacks/cloud-manager/deploy.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add ei https://cdn.einnovator.org/charts
+helm repo update
+
+################################################################################
+# chart
+################################################################################
+STACK="cloud-manager"
+CHART="ei/einnovator-devops"
+CHART_VERSION="1.0.0"
+NAMESPACE="einnovator"
+
+if [ -z "${MP_KUBERNETES}" ]; then
+  # use local version of values.yml
+  ROOT_DIR=$(git rev-parse --show-toplevel)
+  values="$ROOT_DIR/stacks/cloud-manager/values.yml"
+else
+  # use github hosted master version of values.yml
+  values="https://raw.githubusercontent.com/digitalocean/marketplace-kubernetes/master/stacks/cloud-manager/values.yml"
+fi
+
+helm upgrade "$STACK" "$CHART" \
+  --atomic \
+  --install \
+  --namespace "$NAMESPACE" \
```

Comment:
add the following flag to have helm create the namespace

```
--create-namespace \
```

Reply:
The chart is setup with default of 3Gi.
But 2Gi should be enough for minimal workloads (its a java app). (We set it to 3Gi as default because some catalogs that can be imported in the tool, are very big and need some memory to avoid issues.)
The setting --set resources.limits.memory=2Gi --set resources.requests.memory=2Gi could be used for that.
(e.g. If you find that setting to 2Gi is preferred for DO marketplace).

Reply:
I pushed the update to master branch..can you pull it already..or do I have to make an additional action from the the forked repo?!?

Reply:
@paulanunda mention to prompt notification..

Reply:
@paulanunda  pushed the update also to branch pr3-2